File: app/src/main/java/remix/myplayer/lyric/ILrcParser.java
package remix.myplayer.lyric;

import java.io.BufferedReader;
import java.util.List;
import remix.myplayer.lyric.bean.LrcRow;

/**
 * @ClassName
 * @Description
 * @Author Xiaoborui
 * @Date 2016/10/28 09:48
 */

public interface ILrcParser {

  void saveLrcRows(List<LrcRow> lrcRows, String cacheKey, String searchKey);

  List<LrcRow> getLrcRows(BufferedReader bufferedReader, boolean needCache, String cacheKey,
      String searchKey);
}

File: app/src/main/java/remix/myplayer/lyric/ILrcParser.kt
package remix.myplayer.lyric

import remix.myplayer.lyric.bean.LrcRow
import java.io.BufferedReader

/**
 * @ClassName
 * @Description
 * @Author Xiaoborui
 * @Date 2016/10/28 09:48
 */
interface ILrcParser {
  fun saveLrcRows(lrcRows: List<LrcRow>?, cacheKey: String?, searchKey: String?)
  fun getLrcRows(bufferedReader: BufferedReader?, needCache: Boolean, cacheKey: String?,
                 searchKey: String?): List<LrcRow>
}